src/find/matchers/exec.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright 2017 Google Inc.
+//
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+use std::error::Error;
+use std::ffi::OsString;
+use std::io::{Write, stderr};
+use std::path::Path;
+use std::process::Command;
+use walkdir::DirEntry;
+
+use find::matchers::{Matcher, MatcherIO};
+
+pub struct SingleExecMatcher {
+    executable: String,
+    args: Vec<Option<OsString>>,
+    exec_in_parent_dir: bool,
+}
+
+
+
+impl SingleExecMatcher {
+    pub fn new(executable: &str,
+               args: &[&str],
+               exec_in_parent_dir: bool)
+               -> Result<SingleExecMatcher, Box<Error>> {
+
+        let transformed_args = args.iter()
```

Comment:
http://pubs.opengroup.org/onlinepubs/009695399/utilities/find.html says that multiple {}s is implementation defined and GNU find allows multiple {}s. And it's more work to fail with multiple {}s than it is to support it.